mscore/importove.cpp

```diff
@@ -69,188 +69,218 @@
 
 using namespace Ms;
 
+//---------------------------------------------------------
+//   MeasureToTick
+//---------------------------------------------------------
+
 class MeasureToTick {
```

Comment:
Should this class definition, along with the others in this file, perhaps be moved to a header file?

Reply:
`importove.cpp` doesn't have a header file, and I wonder what does it harm to just leave it here? :-)

Reply:
Well, _none_ of the stuff you've changed did any harm before you changed it, so that's beside the point. The style in the rest of the codebase is to put stuff like this in a header file. So, if you're doing a purely stylistic refactor, you may want to consider that :)

Reply:
OK, done.